CODE_OF_CONDUCT.md

```diff
@@ -0,0 +1,52 @@
+# Code of Conduct
+
+## When something happens
+
+If you see a violation of our community's standards of [Conduct](#conduct), follow these steps:
+
+1. Let the person know that what they did is not appropriate and ask them to stop and/or edit their messages.
+2. That person should immediately stop the behavior and correct the issue.
+3. If this doesn’t happen, or if you're uncomfortable speaking up, contact the moderation team.
```

Comment:
how should people get in touch @zkat ?

Reply:
idk. Let's just leave this open-ended for now